crypto/evp/evp_enc.c

```diff
@@ -371,6 +372,19 @@ int EVP_EncryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl,
     return 1;
 }
 
+
+int EVP_EncryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl,
+                      const unsigned char *in, int inl)
+{
+    /* Prevent accidental use of decryption context when encryption */
```

Comment:
grammar: "when encryption"

should be either: "during encryption" or "while encrypting"

(multiple locations)

Reply:
Fixed